### ISSUE TITLE
Visual Indication of Focus for Interface Buttons Too Large on Player Embedded in Dialog 

### ISSUE THREAD

**Issue by jhumb** [ableplayer/ableplayer#134]
On Firefox 38 under Windows 8.1, when embedding able player in a non-modal pop-open dialog, the visual indication of focus for some interface buttons (volume down, preferences, etc.) is too large and encompasses multiple controls. This makes it difficult to tell which control is being selected.

![ableplayer_focus_concern_1](https://cloud.githubusercontent.com/assets/976747/8136895/95ffe32c-110d-11e5-869c-9cb605d2fb96.png)
![ableplayer_focus_concern_2](https://cloud.githubusercontent.com/assets/976747/8136897/99676288-110d-11e5-85b4-7ddf39739fed.png)

**Comment by terrill** [comment]
@jhumb is there an example of that at a public URL?

**Comment by jhumb** [comment]
@terrill Unfortunately, not at the moment. The video was taken down from the test server so updates could be pushed to production. I am working on getting the the video reposted and will share a link.  I will need to contact you outside of GitHub with credentials if a URL can't be made public.

**Comment by jhumb** [comment]
@terrill there is no public URL, but I have a way for you to log into a test account. http://v4.coursenetworking.com/JD17

userid: jd17

Look for an email.  I'm not sure how long the video will be up.

**Comment by blrichwine** [comment]
I ran into this problem when implementing the player in our university's cms template that came with a lot of other css I couldn't control. The short of it was that I added overflow: hidden !important to the buttons in the controller by making the following mod to ableplayer.css:

```css
.able-controller button {
  background: none !important;
  position: relative !important;
  display: inline-block !important;
  border-style: none;
  margin: 0 !important;
  padding: 0 !important;
  color: #fdfdfd;
  min-width: 20px;
  overflow:hidden !important; /* Added to restrain focus indication */
}
```

**Comment by blrichwine** [comment]
Here is a video that demo's the above issue and the css fix for it: https://dl.dropboxusercontent.com/u/4504985/www/enlargedfocusindication.mov

**Comment by terrill** [comment]
Added your fix @blrichwine in d18bb51ad2ff700772717677eb9a329ba39d257d 
I was able to replicate the problem in WordPress and this fixes it there. Closing this issue now, but if anyone encounters it again just reopen and we'll have another look. Thanks!
